lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskStatus.cs

```diff
@@ -132,6 +152,7 @@ public void SetResult(byte[] result)
                         State = TaskState.Done;
                         break;
                 }
```

Comment:
Not sure, this was already here before and should probably be the scope of another JIRA.